bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -141,9 +141,9 @@ <h2 class="good">{{_('Freedom is yours') }}</h2>
       <section class="section-current-platform">
         {# L10n: span's below are for visual formatting and line break. #}
         <h4 class="heading-windows">{{_('Download Firefox <span>for Windows</span>') }}</h4>
-        <h4 class="heading-osx">{{_('Download Firefox <span>for OS X</span>') }}</h4>
+        <h4 class="heading-osx">{{_('Download Firefox <span>for macOS</span>') }}</h4>
         <h4 class="heading-linux">{{_('Download Firefox <span>for Linux</span>') }}</h4>
-
+          
```

Comment:
You want to set your editor to trim whitespaces on save, that would get rid of this diff.

 In Atom that's the package whitespaces, other editors have similar options.